providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/Config.java

```diff
@@ -18,6 +18,7 @@ public final class Config {
 
     static final int DEFAULT_DEADLINE = 500;
     static final int DEFAULT_STREAM_DEADLINE_MS = 10 * 60 * 1000;
+    static final int DEFAULT_STREAM_RETRY_GRACE_PERIOD = 50_000;
```

Comment:
i don't know if this is a good default, pls check

Reply:
I think 50s is an odd choice, mostly just because it's not a fraction or multiple of a minute, but that's a nit.

I wonder if there's any reason is should or should not be the same as the `DEFAULT_STREAM_DEADLINE_MS` ... :thinking: 

I think I'd prefer something more like 30s but I don't have a great reason for that.

Reply:
hehe yeah, this is a bit subjective. i think 50s is nice because it is half of 100s :D but you could also argue that 30s is nice because it's half of 1m :D 

Maybe lets use the same as @aepfli used in the python provider, or at least we should maybe align these default anyway?

Reply:
it is now defined within https://flagd.dev/reference/specifications/providers/#configuration-options  - also the property names etc.